extensions/ql-vscode/src/codeql-cli/cli.ts

```diff
@@ -1737,6 +1737,9 @@ export class CliVersionConstraint {
    */
   public static CLI_VERSION_WITH_TRIM_CACHE = new SemVer("2.15.1");
 
+  public static CLI_VERSION_WITH_EXTENSIBLE_PREDICATE_METADATA_IN_QLX =
+    new SemVer("2.16.1");
```

Comment:
This constant is a bit misleading:
- Extensible predicate metadata is stored in a created qlpack (in `.packinfo` files), not in QLX
- The CLI version that starts generating extensible predicate metadata is 2.16.0, not 2.16.1

Perhaps the constant name we want is something like `CLI_VERSION_WITHOUT_MRVA_EXTENSIBLE_PREDICATE_HACK`?

Reply:
I've renamed the constant and the function to better reflect the details, plus added a few comments.